Rover/AP_ExternalControl_Rover.h

```diff
@@ -0,0 +1,26 @@
+/*
+  external control library for rover
+ */
+#pragma once
+
+#include <AP_ExternalControl/AP_ExternalControl.h>
+
+#if AP_EXTERNAL_CONTROL_ENABLED
+
+class AP_ExternalControl_Rover : public AP_ExternalControl {
+public:
+    /*
+      Set linear velocity and yaw rate. Pass NaN for yaw_rate_rads to not control yaw.
+      Velocity is in earth frame, NED [m/s].
+      Yaw is in earth frame, NED [rad/s].
+     */
+    bool set_linear_velocity_and_yaw_rate(const Vector3f &linear_velocity, float yaw_rate_rads) override;
+private:
+    /*
+      Return true if Copter is ready to handle external control data.
+      Currently checks mode and arm states.
+    */
+    bool ready_for_external_control();
```

Comment:
Could add `WARN_IF_UNUSED` for these two functions

Reply:
Could do. They're not marked as such in the Copter implementation of external control. What's the recommended house style for adding these attributes?

Reply:
I'd add them to Copter, perhaps in a separate PR. It's just additional compile-time-safety, not critical or a blocker.

Reply:
Ok - maybe we could do as a follow up in both cases to keep things consistent.